packages/client/lint-staged.config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
-  '**/*.{js,ts,tsx}': 'eslint --fix',
-  '**/*.{ts,tsx}': () => 'tsc --noEmit -p tsconfig.json'
+  'package/client/**/!(*graphql).{js,ts,tsx}': 'eslint --fix',
+  'package/client/**/!(*grphql).{ts,tsx}': () => 'tsc --noEmit -p tsconfig.json'
```

Comment:
i don't think the tsc needs the negation regex (it's also missing an `a`, so probably isn't doing much 😋 )

Reply:
Oops! Fixed.